web/app/containers/checkout-header/container.js

```
@@ -3,34 +3,39 @@ import React from 'react'
 import Button from 'progressive-web-sdk/dist/components/button'
 import {HeaderBar, HeaderBarTitle} from 'progressive-web-sdk/dist/components/header-bar'
 import {Icon} from 'progressive-web-sdk/dist/components/icon'
+import {isRunningInAstro} from '../../utils/astro-integration'
 
 const CheckoutHeader = function(props) {
     const canSignIn = true
-    return (
-        <header className="t-checkout-header">
-            <HeaderBar className="t-checkout-header__bar">
-                <HeaderBarTitle className="u-flex-none u-padding-start u-text-align-start">
-                    <h2 className="t-checkout-header__title u-heading-family u-text-uppercase">
-                        <span className="u-text-lighter">MERLIN'S</span> CHECKOUT
-                    </h2>
-                </HeaderBarTitle>
+    if (isRunningInAstro) {
+        return null
+    } else {
```

Given that you return in the `if` block, you probably don't need an `else` here.  A common pattern is to "fail fast" and then have the "happy path" continue to the end of the function. Having an `else` here just means you nest the "happy path" one level down... when really you don't need to.

See point #4 in this article: https://blog.codinghorror.com/flattening-arrow-code/  :)